common/services/move.test.js

```diff
@@ -1039,24 +1039,23 @@ describe('Move Service', function () {
       const mockResponse = {
         data: {
           ...mockMove,
-          status: 'cancelled',
         },
       }
       let move
 
       beforeEach(async function () {
-        sinon.stub(apiClient, 'update').resolves(mockResponse)
+        sinon.stub(apiClient, 'post').resolves(mockResponse)
```

Comment:
I'm not sure this will work because the `post` method isn't being called directly. You may have to stub `one` and `all` like the allocations test file.

Reply:
It was being set in a different `forEach` a few lines below, but that was obviously unclear, so I've moved those lines next to this one